web/src/loki-client.ts

```diff
@@ -26,6 +26,19 @@ type HistogramQuerParams = {
   tenant: string;
 };
 
+type LokiTailQueryParams = {
```

Comment:
I notice that `LokiTailQueryParams` isn't used. Could you briefely explain the reason for separating out `LokiTailQueryParams` and `CustomTailQueryParams`?

Reply:
So the LokiTailQueryParms parameters are the ones Loki HTTP API requires and the other parameters are ones that the plugin requires. But you're right it might be less confusing to just make them into a single type.